doc/nrf/libraries/modem/location.rst

```diff
@@ -32,7 +32,7 @@ The supported location methods are as follows:
 
 * GNSS positioning
 
-  * Uses :ref:`gnss_interface` for getting the location.
+  * Uses :ref:`GNSS interface<gnss_interface>` for getting the location.
```

Comment:
```suggestion
  * Uses :ref:`GNSS interface <gnss_interface>` for getting the location.
```
There should be a space here. Applies to all the references.

Reply:
Works both with or without the space.

Reply:
I was referring to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#internal_links. :) Please leave it as is .